frontend/packages/operator-lifecycle-manager/src/components/operator-group.tsx

```diff
@@ -12,56 +12,61 @@ import { AsyncComponent } from '@console/internal/components/utils/async';
 import { OperatorGroupKind, SubscriptionKind, InstallModeType } from '../types';
+import { useTranslation } from 'react-i18next';
 
 export const targetNamespacesFor = (obj: K8sResourceKind) =>
-  _.get(obj, ['metadata', 'annotations', 'olm.targetNamespaces']);
+  obj?.metadata?.annotations?.['olm.targetNamespaces'];
 export const operatorNamespaceFor = (obj: K8sResourceKind) =>
-  _.get(obj, ['metadata', 'annotations', 'olm.operatorNamespace']);
+  obj?.metadata?.annotations?.['olm.operatorNamespace'];
 export const operatorGroupFor = (obj: K8sResourceKind) =>
-  _.get(obj, ['metadata', 'annotations', 'olm.operatorGroup']);
-
-export const NoOperatorGroupMsg: React.SFC = () => (
-  <MsgBox
-    title="Namespace Not Enabled"
-    detail={
-      <p>
-        The Operator Lifecycle Manager will not watch this namespace because it is not configured
-        with an OperatorGroup.{' '}
-        <Link to={`/ns/${getActiveNamespace()}/${referenceForModel(OperatorGroupModel)}/~new`}>
-          Create one here.
-        </Link>
-      </p>
-    }
-  />
-);
-
-type RequireOperatorGroupProps = {
-  operatorGroup: { loaded: boolean; data?: OperatorGroupKind[] };
+  obj?.metadata?.annotations?.['olm.operatorGroup'];
+
+export const NoOperatorGroupMsg: React.FC = () => {
+  const { t } = useTranslation();
+  return (
+    <MsgBox
+      title="Namespace Not Enabled"
+      detail={
+        <p>
+          {t(
+            'olm~The Operator Lifecycle Manager will not watch this namespace because it is not configured with an OperatorGroup.',
+          )}
+          &nbsp;
+          <Link to={`/ns/${getActiveNamespace()}/${referenceForModel(OperatorGroupModel)}/~new`}>
+            {t('olm~Create one here.')}
+          </Link>
+        </p>
+      }
+    />
+  );
 };
 
-export const OperatorGroupSelector: React.SFC<OperatorGroupSelectorProps> = (props) => (
-  <AsyncComponent
-    loader={() =>
-      import('@console/internal/components/utils/list-dropdown').then((m) => m.ListDropdown)
-    }
-    onChange={
-      props.onChange ||
-      function() {
-        return null;
+export const OperatorGroupSelector: React.FC<OperatorGroupSelectorProps> = (props) => {
+  const { t } = useTranslation();
+  return (
+    <AsyncComponent
+      loader={() =>
+        import('@console/internal/components/utils/list-dropdown').then((m) => m.ListDropdown)
       }
-    }
-    desc="Operator Groups"
-    placeholder="Select Operator Group"
-    selectedKeyKind={referenceForModel(OperatorGroupModel)}
-    dataFilter={props.dataFilter}
-    resources={[
-      {
-        kind: referenceForModel(OperatorGroupModel),
-        fieldSelector: `metadata.name!=${props.excludeName}`,
-      },
-    ]}
-  />
-);
+      onChange={
+        props.onChange ||
+        function() {
+          return null;
+        }
+      }
+      desc={OperatorGroupModel.plural}
+      placeholder={t('olm~Select {{item}}', { item: OperatorGroupModel.label })}
```

Comment:
Externalize `OperatorGroup` as part of the message instead of using model.plural